password_security/models/res_users.py

```diff
@@ -156,3 +156,20 @@ def _set_encrypted_password(self, encrypted):
                 'password_crypt': encrypted,
             })],
         })
+
+    # Dirty fix because this was failing when any theme was installed
+    # I think this should be a fix in the core.
+
+    @api.multi
+    def _is_system(self):
+        if not self:
+            return False
+        return super(ResUsers, self)._is_system()
+
+    @api.multi
+    def _is_superuser(self):
+        if not self:
+            return False
+        return super(ResUsers, self)._is_superuser()
+
+    ############ End of dirty fix ###############
```

Comment:
yes, that was the orginal solution that I proposed, but @oscarolar and @nhomar point me to this [fix](https://github.com/Vauxoo/server-tools/commit/d409c85dda6eb46209e04e8e5f10615737a299e0)

Here is the pr to odoo/odoo https://github.com/odoo/odoo/pull/16147